troi/acousticbrainz/annoy.py

```diff
@@ -34,12 +34,12 @@ def read(self, inputs, debug=False):
         url = self.SERVER_URL + self.metric + "/" + self.mbid
         r = requests.get(url)
         if r.status_code != 200:
-            r.raise_for_status()
+            raise PipelineError("Cannot fetch annoy similarities from AcousticBrainz: HTTP code %d" % r.status_code)
```

Comment:
I think this is an OK way of doing it. raise_for_status just checks the status code and raises an exception if it's not 200. This means the if/raise here is redundant.
It should be no different here to do if/raise PiplineError or try/raise_for_status/except/raise PipelineError, and the first one is less lines, so let's stay with that